### ISSUE TITLE
[Buttons] Disabled buttons lack sufficient contrast

### ISSUE THREAD

**Issue by ianegordon** [material-components/material-components-ios#3167]
b/74073793
b/74075087

Contrast seems really low, especially on disabled buttons.

Fix should ensure behavior in smart inversion also meets contrast requirements.

**Comment by romoore** [comment]
These are bugs for a specific set of color values on an internal client. They cannot be fixed in MDC.

**Comment by ianegordon** [comment]
![screen shot 2018-03-28 at 16 10 33](https://user-images.githubusercontent.com/1121006/38053612-947c71aa-32a2-11e8-9640-ec9345443a68.png)

**Comment by romoore** [comment]
Those colors are chosen for the catalog and so aren't the internal b/ issues. Do you want to update this Issue to correct the catalog colors?

**Comment by romoore** [comment]
Both internal issues are resolved.
